app/ui/src/app/integrations/edit-page/step-configure/data-mapper/data-mapper-host.component.ts

```diff
@@ -107,6 +108,7 @@ export class DataMapperHostComponent extends FlowPage implements OnInit {
     // TODO not sure what to do for `none` or `any` here
     switch (kind) {
       case 'java':
+        this.initializedCountDown += 1;
```

Comment:
Purely in the interest of maintenance maybe wrap this in a function like `addInitializationTask()`